public/app/features/plugins/admin/pages/Browse.tsx

```diff
@@ -132,6 +147,11 @@ const getStyles = (theme: GrafanaTheme2) => ({
   listWrap: css`
     margin-top: ${theme.spacing(2)};
   `,
+  displayAs: css`
+    svg {
+      margin-right: 0;
```

Comment:
We have some styles in the `<Icon/>` that don't play nicely with `RadioButtonGroup`?